src/handleAction.js

```diff
@@ -2,27 +2,26 @@ function isFunction(val) {
   return typeof val === 'function';
 }
 
+function coerceReducer(reducer) {
+  return isFunction(reducer) ? reducer : state => state;
```

Comment:
This isn't a blocking concern, but I still wish we were more intentional about the argument type passed in. 

`null` and `undefined` seem reasonable to coerce to a default, but `1`, `{}`, `[]`, `true`, `false` seems like a misuse of the API that we should throw `TypeError` on.

IMO, the most ideal situation would be to not conflate `null` and `undefined`, and just pick one to coerce, preferably `undefined`, so we could use default argument syntax like https://github.com/yangmillstheory/redux-standard-actions/blob/master/src/actionReducer.js#L27.